embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/LogSink.kt

```diff
@@ -0,0 +1,27 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.opentelemetry.sdk.common.CompletableResultCode
+import io.opentelemetry.sdk.logs.data.LogRecordData
+
+/**
+ * A service that stores exported logs and provides access to them so they
+ * can be sent off-device at the appropriate cadence.
+ */
+internal interface LogSink {
+
+    /**
+     * Stores Logs. Implementations must support concurrent invocations.
+     */
+    fun storeLogs(logs: List<LogRecordData>): CompletableResultCode
+
+    /**
+     * Returns the list of currently stored Logs.
+     */
+    fun logs(): List<EmbraceLogRecordData>
```

Comment:
nit: could this be named `completedLogs`, similar to how `SpanSink` works?

Reply:
Renamed.